lib/reporter/magellan-i18n-reporter.js

```diff
@@ -0,0 +1,206 @@
+const BaseReporter = require( 'testarmada-magellan' ).Reporter;
```

Comment:
Yeah, I originally was thinking about making more changes to the reporter but given that it's just this change I'll go ahead and do that.